internal/blockchain/v0/pool.go

```diff
@@ -428,6 +443,20 @@ func (pool *BlockPool) debug() string {
 	return str
 }
 
+func (pool *BlockPool) targetSyncBlocks() int64 {
```

Comment:
This is not the target height rather the amount of blocks remaining to be synced right?

Reply:
yes, it's the block height the node wants to sync to. Should be the same as max peer height.

Reply:
But here we're returning:
```
return pool.maxPeerHeight - pool.startHeight + 1
```
Is that right?

Reply:
Oh, sorry, it's the total amount of sync blocks that will need to sync when the reactor starting to sync.

Reply:
so the remaining blocks will be the returns of `targetSyncBlocks` - current synced blocks since started. and then we divide by the lastSyncRate, so we can know the remaining sync time.